content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,277 @@
+
+## CTest
+
+CTest is similar to other test frameworks like GoogleTest, PyTest, JUnit, or NUnit; though it uses a higher-level generic interface. CTest registers each test as a string of command-line arguments, and reports on whether invoking those arguments succeeded (returned `0`) without hanging. CTest can also coordinate running tests in parallel processes. Unlike other test runners, CTest is programming language agnostic by interfacing with the OS shell. However this means a notable feature is absent from CTest: other test frameworks typically provide a lower-level interface which directly invokes functions inside a specific programming language. To provide this function-level execution, O3DE uses CTest to invoke lower-level test runners.
+
+O3DE provides wrapper code to help register tests for frameworks such as GoogleTest and PyTest into CTest. Along with the provided test-tools, these wrappers enable writing tests which can run on any operating system. XML files are generated to track the low-level results not reported to CTest, and artifacts are saved such as log output and crash dumps.
+
+![CTest calls other test runners](/images/user-guide/testing/getting-started/ctest_to_runners.png)
+
+In short, all tests in O3DE use CTest as a high-level test coordinator. When executed, CTest calls other lower-level test tools and then reports on their success.
+
+### Starting CTest
+
+CTest expects its working directory to be a CMake build directory, so be sure to first navigate a terminal to this directory. Then use `-C` to select a build configuration one you have already built locally:
+
+* **Windows**:
+    ```shell
+    cd <local_path_to>\\o3de\\build\\<build_folder>
+    ctest -C <build_configuration>
+    ```
+
+* **Linux**:
+    ```shell
+    cd <local_path_to>/o3de/build/<build_folder>
+    ctest -C <build_configuration>
+    ```
+
+{{< caution >}}
+Without a filter the command will run every registered test, and likely result in multiple hours of test execution! If you ever want to stop CTest, send an interrupt signal by selecting the terminal and pressing `CTRL+C`.
+{{< /caution >}}
+
+CTest can also run a subset of labeled test suites with the `-L` argument. These test suites should be contained in parenthesis (`(..)`) and have their names separated with a `|` character. The following examples demonstrate this syntax for running the Main and Smoke suites for a `profile` build on Windows or Linux:
+
+* **Windows**:
+    ```shell
+    cd C:\\o3de\\build\\vs_2019
+    ctest -C profile -L "(SUITE_smoke|SUITE_main)"
+    ```
+* **Linux**:
+    ```shell
+    cd user/github/o3de/build/linux
+    ctest -C profile -L "(SUITE_smoke|SUITE_main)"
+    ```
+
+The Smoke and Main suites execute against pull requests in the AutomatedReview pipeline, and both are recommended for local verification. These suites are required to execute relatively fast, and must not intermittently fail.
```

Comment:
```suggestion
It's recommended that you verify your tests with the Main and Smoke suite on your local machine, as these tests will be executed on your pull request (PR), as part of the Automated Review pipeline in the `o3de` repository's PR workflow. For a successful PR, these suites must execute relatively fast and must not intermittently fail on your code. 
```
Rephrased some of this paragraph to clarify what the user is recommended to do (in "active" voice) and why.

Reply:
Some of the direction of implication has been changed. I'll rewrite to make this clearer.

Reply:
```suggestion
It's recommended that you verify your tests with the Main and Smoke suite on your local machine. These tests will be executed on your pull request (PR), as part of the Automated Review pipeline in the `o3de` repository's PR workflow. Both of these suites must execute relatively fast and must not intermittently fail, and are an easy way to prove your change did not break other features. 
```

Reply:
Great! Commit it